docs/guides/software-lifecycle/security/security-scanning/README.md

```diff
@@ -0,0 +1,238 @@
+# Security Scanning
+
+<pre align="center">Guide to code scanning for security issues.</pre>
+
+## Introduction
+
+**Background**: Software security is critical in modern systems with application code at its root. Identifying and addressing vulnerabilities rapidly mitigates risk and limits the potential surface area of attacks. We recommend [NASA's SCRUB platform](https://github.com/nasa/scrub) to manage code scanning by identifying, orchestrating and aggregating security information. SCRUB's GitHub implementation wraps [CodeQL](https://codeql.github.com/) results into compact, curated reports that highlight security assessments and are suitable for ingestion by automated reporting tools. A small configuration is appended to an existing CodeQL configuration (`codeql-config.yml` file) that specifies security analyses and reporting properties.  
+
+**Use Cases**:
+- Finding and mitigating security risks in code, such as:
+  - unknown
```

Comment:
Great progress so far! I think filling this section out will really help readers understand what our guide could help with and what it couldn't.

By the way - do you think there's overlap or synergy with [this ticket](https://github.com/NASA-AMMOS/slim/issues/109)? We have [a PR](https://github.com/NASA-AMMOS/slim/pull/116) for that ticket still awaiting finalization. Wonder if it makes sense to bring the two together?

Reply:
> Do you think there's overlap or synergy with [this ticket](https://github.com/NASA-AMMOS/slim/issues/109)? We have [a PR](https://github.com/NASA-AMMOS/slim/pull/116) for that ticket still awaiting finalization. Wonder if it makes sense to bring the two together?

I feel they are separate concerns. This is the actual implementation. That feels more like a reference guide that deals with a number of concerns of which code security scanning would be one. I think the result of that ticket -- a Security Guide -- should reference this implementation.

Reply:
Okay, I'm resolving this comment. I renamed the [referenced ticket](https://github.com/NASA-AMMOS/slim/issues/109) to reflect a ticket to create a Security Reference Architecture.